src/getCoinCombination.test.js

```diff
@@ -7,4 +7,37 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should be non-negative input integer', () => {
```

Comment:
The test description 'should be non-negative input integer' could be more precise. Consider rephrasing it to something like 'should throw an error for negative input'.